content/YubiHSM2/Usage_Guides/YubiHSM_2_for_Microsoft_SQL_Server_Deployment_Guide--Enabling_Always_Encrypted_with_YubiHSM_2/Getting_Help_and_Further_Reading.adoc

```diff
@@ -0,0 +1,28 @@
+== Getting Help and Further Reading
+
+Should you require assistance when using this guide to deploy YubiHSM 2 with Microsoft SQL Server Always Encrypted, start by referencing the product documentation and currently known issues:
+
+link:https://developers.yubico.com/[Yubico Developers website]
+
+link:https://support.yubico.com/support/home[Yubico Support]
+
+link:https://developers.yubico.com/YubiHSM2/[Product documentation]
+
+link:https://developers.yubico.com/YubiHSM2/Releases/Known_issues.adoc[Known issues and limitations]
+
+If you need additional help, contact Yubico directly by filling in a ticket at link:https://support.yubico.com/support/home[Yubico’s support site].
+
+In addition to the Yubico web sites listed above, Microsoft has published the following articles on SQL Server Always Encrypted:
+
+link:link:https://support.yubico.com/support/home[Always Encrypted (Database Engine)]
```

Comment:
shouldn't this point to a microsoft page? also it has double link: at the start

Reply:
It should DEFINITELY point to a Microsoft page.

Reply:
https://github.com/Yubico/developers.yubico.com/pull/250#discussion_r357091797--also made links to dyc relative